bigquery_etl/format_sql/formatter.py

```diff
@@ -281,7 +295,9 @@ def inline_block_format(tokens, max_line_length=100):
                 line_length += line.inline_length
                 if line_length > max_line_length:
                     break
-                if line.indent_level <= indent_level:
+                if line.starts_with_closing_bracket:
+                    open_brackets -= 1
+                if open_brackets == 0:
```

Comment:
This change was necessary because the "hanging indent" for `WHEN`/`THEN`/`ELSE` values means we can't rely on the indent level to determine if we're done with a bracket block.